docs/uptime/uptime.md

```diff
@@ -0,0 +1,289 @@
+# About Uptime
+
+"Is my website up?" is the most basic question that any website owner or operator needs to answer at any given time. During a time-sensitive incident, the answer to this question is essential for the next steps that need to be taken.
+
+This makes it the single most important visualization that we provide in the Synthetic Monitoring application. It is crucial that it is as accurate as it can be so that users can feel confident in the data that they are seeing.
+
+If uptime has any degree of inaccuracy, it can lead to a false sense of security or panic and frustration, which reduces confidence in our product and perceived value.
+
+## Use cases
+
+1. **"Is my site up right now?"** - Most daily use-cases of Synthetic Monitoring will be over smaller time ranges of a few hours. Our default is 3 hours but users might adjust the time range to 1 hour or 24 hours.
+2. **"I have a fortnightly SLA to report"** - Reporting uptime metrics for weekly / bi-weekly / monthly SLAs or other business metrics at regularly occurring meetings. In this case, a moderate time period, such as a week or a month is a likely scenario.
+3. **"Is our hosting provider reliable?"** - To assess whether more strategic decisions are warranted and to paint a holistic picture, it is equally valid to query much longer time periods, such as a quarter or even a year.
+4. **"We had an incident. I need to know exactly what happened."** - a query of just a few minutes is very likely in the case of an incident that requires a deep level of analysis when problems began and ended so the root cause can be established.
+
+When we consider all of these use cases, we have to cater for an accurate uptime metric for as little as a minute up to one year. This is a **huge** range of time periods and we need to be able to accurately calculate uptime for all of them.
+
+## How we collect and query data
+
+![](./images/sm-architecture.png)
+
+[A more detailed explanation of the Synthetic Monitoring architecture in our on-call training.](https://docs.google.com/presentation/d/1jWuv2VwVFIsdyXq4JQMlhmM6txHXLyHImYpnl50V5XQ/edit#slide=id.g308ff111cb5_0_35).
```

Comment:
Flagging that this is an internal-only link in a public repo, so the general public may click thinking they can access.  It might be worth adding "(internal-only)" or something to set expectations this won't be externally visible. 
```suggestion
[A more detailed explanation of the Synthetic Monitoring architecture in our on-call training (Grafana Labs internal-only).](https://docs.google.com/presentation/d/1jWuv2VwVFIsdyXq4JQMlhmM6txHXLyHImYpnl50V5XQ/edit#slide=id.g308ff111cb5_0_35).
```